libraries/script-engine/src/ScriptEngine.cpp

```diff
@@ -130,6 +130,11 @@ static QScriptValue debugPrint(QScriptContext* context, QScriptEngine* engine) {
     // This message was sent by one of our script engines, let's try to see if we can find the source.
     // Note that the first entry in the backtrace should be "print" and is somewhat useless to us
     AbstractLoggerInterface* loggerInterface = AbstractLoggerInterface::get();
+    if (!loggerInterface) {
+        qCDebug(scriptengine_script, "%s", qUtf8Printable(message));
+        return QScriptValue();
+    }
+
     if (loggerInterface->showSourceDebugging()) {
```

Comment:
```suggestion
    if (loggerInterface && loggerInterface->showSourceDebugging()) {
```
This should work, I think, and will `scriptEngine->print()` the message and `qCDebug()` the message with the filename.